dev_guide/builds.adoc

```diff
@@ -650,25 +652,39 @@ running in OpenShift.
 +
 In this case, the `*sourceStrategy*` definition is consuming the `latest` tag of
 the image stream named `ruby-20-centos7` located within this namespace.
-
-. Define an image change trigger:
++
+Add one or more image change triggers to the build config spec:
 +
 ====
 
 ----
 {
-  "type": "imageChange",
+  "type": "imageChange", <1>
   "imageChange": {}
 }
+{
+  "type": "imagechange", <2>
+  "imageChange": {
+     "from": {
+       "kind": "ImageStreamTag",
+       "name": "custom-image:latest"
+     }
+  }
+}
 ----
+
+<1> An image change trigger that monitors the `*ImageStream*` and
+`*Tag*` as defined by the strategy's `*from*` field. The `*imageChange*` part
+must be empty.
+<2> An image change trigger that monitors an arbitrary image stream. The `*imageChange*`
+part in this case must include a `*from*` field that references the `*ImageStreamTag*` to monitor.
 ====
-+
-This defines an image change trigger that monitors the `*ImageStream*` and
-`*Tag*` as defined by the strategy's `*from*` field. When a change occurs, a new
-build is triggered and is supplied with an immutable Docker tag that points to
-the new image that was just created. This new image will be used by the strategy
+
+When a new image with the corresponding tag is added to either image stream, a new build is
+triggered. The generated build is supplied with an immutable Docker tag
+that points to the latest image that corresponds to that tag. This new image will be used by the strategy
```

Comment:
the new image (the one that triggered the build) won't necessarily be used by the strategy..  only the default trigger has that behavior.  the others just run a build.